OasisDEX/oasis-borrow
# Don't redirect to stheth/eth position if the position already has things in it

<img width="989" alt="image" src="https://user-images.githubusercontent.com/3783198/209147472-5d720cc2-5a82-4fcf-8d2c-a34c635b8a85.png">

  
## How to test 🧪

- create a wbtc/usdc Aave position not using maker, using DS proxy
- go to create a new steth/eth earn position from product card
- expected: you see the warning and are not redirected
